packages/coverage-c8/src/provider.ts

```diff
@@ -193,3 +193,27 @@ export class C8CoverageProvider extends BaseCoverageProvider implements Coverage
     }
   }
 }
+
+function removeViteHelpersFromSourceMaps(source: string | undefined, map: EncodedSourceMap): EncodedSourceMap {
+  if (!source?.includes('__vite_ssr_exports__'))
+    return map
+
+  const exportIndexes = source.split('\n').map((line, index) => {
+    if (/Object\.defineProperty\(__vite_ssr_exports__/.test(line))
+      return index
+    return -1
+  }).filter(index => index !== -1)
+
+  if (exportIndexes.length === 0)
+    return map
+
+  const mappings = map.mappings.split(';')
+  exportIndexes.forEach((index) => {
+    mappings[index] = ''
+  })
+
+  return {
+    ...map,
+    mappings: mappings.join(';'),
+  }
+}
```

Comment:
This functions looks for lines where `__vite_ssr_exports__` is appended with new getters and removes those lines from the source maps. It's hacky but it shows what should be done.

Any ideas what would the correct way to do this? I don't think `magic-string` supports modifying existing source maps - it simply generates new ones. And I guess there is no way to tell Vite not to include these lines in source maps. 

Also it would be great if we could do this directly in rollup/vite plugins but I don't think Vite exports are available there even in `enforce: 'post'`.

Reply:
I picked only the needed `remapping` from `combineSourcemaps`: https://github.com/ampproject/remapping#multiple-transformations-of-a-file

I checked all files of `test/coverage-test` and it seems to work well. Vue had some issues but eventually got it working as well. Source maps are fun - said no one ever. 🙃